radio-packet-format/sections/altitude.tex

```diff
@@ -0,0 +1,29 @@
+\subsection{Altitude}
+
+The altitude block is used to convey the altitude of the rocket. The format of the altitude block payload is described
+in Figure \ref{format:telem-altitude}.
+
+\begin{figure}[H]
+    \centering
+    \begin{bytefield}[bitwidth=0.03\linewidth]{16}
+        \bitheader{0-15} \\
+        \wordbox{1}{Measurement Time} \\
+        \wordbox{2}{Altitude} \\
+    \end{bytefield}
+    \caption{Altitude Data Payload Format}
+    \label{format:telem-altitude}
+\end{figure}
+
+\blocktimestampexp
+
+\paragraph{Altitude}
+
+The calculated altitude in units of millimetres. This field is a signed 32 bit integer in two's complement format.
+
+If the block header specifies the type for altitude above sea level, this measurement should be interpreted as
+millimetres above sea level.
+
+If the block header specifies the type for altitude above launch height, this measurement should be interpreted as
+millimetres from the starting height of the rocket.
```

Comment:
Should this say "altitude above launch level"

Reply:
Don't think so? Launch height being the height he rocket lifts off from. Do you think this could be clarified?

Reply:
I was thinking like "ground level" might be more clear but honestly this is a nit and not that important.

Reply:
I can add (ground level) in brackets! Nitpicking is good esp if it helps make the spec more clear.